plugin.yaml

```diff
@@ -1639,6 +1639,8 @@ node_types:
         start: azure.cloudify_azure.resources.deployment.pull
         delete: azure.cloudify_azure.resources.deployment.delete
         pull: azure.cloudify_azure.resources.deployment.pull
+        update: azure.cloudify_azure.resources.deployment.update
+
```

Comment:
So params are accepted from update operation, because params is in the signature.
But what happens to kwargs?

Perhaps we should try updaing handle task:

```
def handle_task(resource,
                resource_group_name,
                name,
                parent_name=None,
                resource_task=None,
                additional_params=None,
                **kwargs):
    """
    :param resource: A AzureResource object from azure_sdk package.
    :param resource_group_name: An Azure resource group name.
    :param name: The name of the resource.
    :param parent_name: The name of the resource's parent resource,
        if applicable.
    :param resource_task: The name of the task, e.g. delete, get, that we wish
        to call and to handle.
    :param additional_params: Task parameters that may or may not be used.
    :return:
    """
    if not resource_task:
        raise RuntimeError('No resource task name provided.')
    task = getattr(resource, resource_task)
    args = [resource_group_name]
    if parent_name:
        args.append(parent_name)
    if name:
        args.append(name)
    if additional_params:
        args.append(additional_params)
    try:
        return task(*args, **kwargs)
    except (CloudError, ResourceNotFoundError) as e:
        return e
```